Revert-tvOS/Sources/CollectionDataSource.swift

```diff
@@ -1,60 +1,71 @@
-//
-//  Copyright © 2016 Itty Bitty Apps. All rights reserved.
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
```

Comment:
Okey doke — I'm going to point this nit pick out and then go do a full (proper) review, because this ain't it, Chief 😉  

You don't every need to update the copyright of a source code file. The year in here is an indication of when the copyright is valid _from_ (ie. the creation date of the file). It's honestly nonsense, and I'd prefer we stop including it, but the "Copyright/All rights reserved" to IBA is the important bit here - the rest can probably be derived from metadata. 

I would recommend changing this copyright header (and any others) back to their original state. I'm happy for the white space changes to stay, if they're consistent with the other files.

Reply:
Ah sorry for that! I did not mean to rewrite the history.

So what happened was, I created a new class named `NewCollectionDataSource` which has the new copyright source by default.
Then After refactoring, I delete the old `CollectionDataSource` and rename `NewCollectionDataSource` -> `CollectionDataSource`.
I think, Git considers it as me making changes to the old file.

I definitely agree with you that the copyright year is not meant to be changed. But I definitely think we still should include the year when it was valid from.

Will revert the copyright header in the next commit 👍.

Reply:
Brilliant, ta!